src/app/dim-ui/Select.tsx

```diff
@@ -91,29 +136,31 @@ export default function Select<T>({
         {...getMenuProps({ ref: menuRef })}
         className={clsx(styles.menu, { [styles.open]: isOpen })}
       >
-        {isOpen &&
-          items.map(
-            (item, index) =>
-              !(hideSelected && item.value === value) && (
-                <div
-                  className={clsx(styles.menuItem, {
-                    [styles.highlighted]: highlightedIndex === index,
-                    highlighted: highlightedIndex === index,
-                    [styles.disabled]: item.disabled,
-                  })}
-                  key={item.key}
-                  {...getItemProps({
-                    item,
-                    index,
-                    onClick: (e: any) => {
-                      e.nativeEvent.preventDownshiftDefault = item.disabled;
-                    },
-                  })}
-                >
-                  {item.content}
-                </div>
-              )
-          )}
+        <div style={dropdownStyle}>
```

Comment:
I didn't want to mess with how popper works so I just added another div and put the styles on that.